app/src/main/java/org/oppia/app/profile/ProfileChooserListener.kt

```diff
@@ -0,0 +1,7 @@
+package org.oppia.app.profile
+
+import androidx.databinding.ObservableField
+
+interface ProfileChooserListener {
```

Comment:
Is this file actually used? If not, please remove. Ditto elsewhere--please remove any unused classes/symbols, etc.

Reply:
Okay!